src/main/SubtitleListEditor.tsx

```diff
@@ -402,54 +396,90 @@ const SubtitleListSegment = React.memo((props: subtitleListSegmentProps) => {
             tooltipAria={t("subtitleList.endTime-tooltip-aria")+": " + convertMsToReadableString(cue.endTime)}
           />
         </div>
-
         <div css={functionButtonAreaStyle} className="functionButtonAreaStyle">
-          <div css={[basicButtonStyle(theme), addSegmentButtonStyle]}
-            role="button" tabIndex={0}
-            title={t("subtitleList.addSegmentAbove")}
-            arial-label={t("subtitleList.addSegmentAbove")}
+          <FunctionButton
+            style={[basicButtonStyle(theme)]}
```

Comment:
Seems weird to have component to which you can pass a style, which is then partially overwritten in the component. I am curious what your rationale for that choice was? Do you expect different `basicButtonStyles` in the future?

Reply:
I'm no longer quite sure what I was thinking and why I did it the way I did it. But where you mention it, you're right, it doesn't make any sense. I will remove it and add it directly to the component, without the 'basicButtonStyle' e.g. the icons in the buttons are no longer centered and there is no hover transformation anymore.